### PR TITLE
Fixed lti_version LTI parameter from 1.1.1 to LTI-1p0

### PR DESCRIPTION
**Pull request by uncleempty**
This fix doesn't belong to any ticket.

Reason to do this fix is the point that sometime ago while implementing LTI 1.3 support, accidentally we gone with the wrong version string for LTI 1.1 when communicating with TAO.
Indeed, in terms of SR - we don't care and it can be anything. BUT, TAO side awaits for exactly `LTI-1p0` in case we are using LTI 1.1

### CODE REVIEW

File: tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php [hunk 1]
```diff
@@ -234,7 +234,7 @@ public function testItReturnsLtiLinkAndUpdatedAssignmentAndAttemptsCountWithUser
 
         $authenticationResponse = $this->logInAs($user, $this->kernelBrowser);
 
-        $_ENV['LTI_VERSION'] = LtiRequest::LTI_VERSION_1P1;
```
Comment (Skyld) [inline tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php:237]: Why did you do this change?

File: tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php [hunk 2]
```diff
@@ -296,7 +296,7 @@ public function testItReturnsLtiLinkAndUpdatedAssignmentAndAttemptsCountWithUser
...
```
Comment (Skyld) [inline tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php:299]: Why did you do this change?

File: tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php [hunk 3]
```diff
@@ -353,7 +353,7 @@ public function testItReturnsLoadBalancedLtiLinkAndUpdatedAssignmentStateToStart
     {
         $initialLtiLaunchPresentationLocale = $_ENV['LTI_LAUNCH_PRESENTATION_LOCALE'];
 
-        $_ENV['LTI_VERSION'] = LtiRequest::LTI_VERSION_1P1;
```
Comment (Skyld) [inline tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php:356]: Why did you do this change?

**Review by Skyld — CHANGES_REQUESTED** [review]

**Review by Skyld — DISMISSED** [review]
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes

**Review by Skyld — APPROVED** [review]
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes

**Review by andreluizmachado — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes (if applicable)
- [x] Documentation is updated according to changes (if applicable)

**Pull request merged**